source/_integrations/speedtestdotnet.markdown

```diff
@@ -1,50 +1,39 @@
 ---
-title: Speedtest.net
+title: SpeedTest.net
 description: How to integrate Speedtest.net within Home Assistant.
 ha_category:
   - System Monitor
   - Sensor
 ha_release: 0.13
 ha_iot_class: Cloud Polling
+ha_config_flow: true
 ha_codeowners:
   - '@rohankapoorcom'
+  - '@rngrbm87'
 ha_domain: speedtestdotnet
 ---
 
 The `speedtestdotnet` integration uses the [Speedtest.net](https://speedtest.net/) web service to measure network bandwidth performance.
 
-Enabling this integration will automatically create Speedtest.net sensors for the monitored conditions (below).
-
 By default, a speed test will be run every hour. The user can change the update frequency in the configuration by defining the `scan_interval` for a speed test to run.
 
 Most Speedtest.net servers require TCP port 8080 outbound to function. Without this port open you may experience significant delays or no results at all. See note on their [help page](https://www.speedtest.net/help).
 
 ## Configuration
 
-For the `server_id` check the list of [available servers](https://speedtest.net/speedtest-servers.php).
+Set up the integration through **Configuration -> Integrations -> SpeedTest.net**. Once configured you can select the server to run the test against from the options menu. You can also change the update interval and optionaly disable auto update.
 
-To add Speedtest.net sensors to your installation, add the following to your `configuration.yaml` file:
 
-Once per hour, on the hour (default):
+To import the configuration from `configuration.yaml` refer to the below example. Make sure to remove `monitored_condiditions` as they are now automatically added to Home Assistant.
```

Comment:
The monitored conditions remark do not apply to the current documentation and should be part of the breaking changes section in the release notes. Please remove that remark from the documentation.